ruby/syntax/lexical/VarName.sdf3

```diff
@@ -1,13 +1,13 @@
 module VarName
 
 imports
-	Common
+	Identifier
 	Global
 
 lexical syntax
 
 	VarName = Global
-	VarName = "@"ID // TO DO : no spacing allowed.
+	VarName = "@" ID
```

Comment:
For pretty printing you would not want the space right?

Reply:
This in lexical syntax so this is different from what you think. Here it does not make any sense to include layout, since lexical syntax is meant to describe the tokens that the CFG works with and optionally separates by layout (AFAIK). So that space character discussed above does not mean anything. In fact, without the space the rule seemed to break. And above all, with space it is more readable. The spaces in rules like this actually serve that purpose I believe; to separate character classes and symbols et cetera. If you would want to support spaces as part of the token in your lexical syntax rule, you would explicitly add it like this:

```
"@" " " ID
```

Or, maybe to support more general whitespace, something like (don't know if it works):

```
"@" LAYOUT* ID
```

Reply:
Alright, clear.